packages/infrastructure/mongodb_atlas_identity_provider/main.tf

```diff
@@ -0,0 +1,79 @@
+terraform {
+  required_providers {
+    mongodbatlas = {
+      source  = "mongodb/mongodbatlas"
+      version = "1.26.1"
+    }
+
+    authentik = {
+      source  = "goauthentik/authentik"
+      version = "2024.8.4"
+    }
+  }
+}
+
+locals {
+  default_role_mappings = {
+    "superusers"            = ["ORG_OWNER"]
+    "billing_admins"        = ["ORG_BILLING_ADMIN"]
+    "privileged_engineers"  = ["ORG_GROUP_CREATOR", "ORG_READ_ONLY"]
```

Comment:
I think we said this was going to be ORG_OWNER. Additionally, update the docs if you change it here.